### ISSUE TITLE
Wrong access rights of pcmdi_metrics-v1.1.1-py2.7.egg-info in 2.8.0!

### ISSUE THREAD

**Issue by jypeter** [PCMDI/pcmdi_metrics#496]
@doutriaux1 and @durack1  with my usual luck, I have found an unusual problem...

I have installed **PMP** in 2.8.0, just in case, with
`conda install -n cdatm14 -c conda-forge -c pcmdi pcmdi_metrics`

I have not used PMP, so I assume it is OK, but it has the following **weird side effect**: if another person uses my **shared CDAT installation**, AND wants to use ipython, then there is a huge traceback due to the fact that **pcmdi_metrics-v1.1.1-py2.7.egg-info can be read only by the owner** (the person who installed cdat, i.e me)

```
(cdatm14) bash-4.1$ ls -l /home/share/unix_files/cdat/miniconda2/envs/cdatm14/lib/python2.7/site-packages/pcmdi_metrics-v1.1.1-py2.7.egg-info
-rw------- 2 jypeter lsce 228 21 juil. 2016 /home/share/unix_files/cdat/miniconda2/envs/cdatm14/lib/python2.7/site-packages/pcmdi_metrics-v1.1.1-py2.7.egg-info
```

I have checked the content of envs/cdatm14/lib/python2.7/site-packages/ on 2 different servers where I have installed 2.8.0 and pcmdi_metrics-v1.1.1-py2.7.egg-info is the only file that is not readable by group+other.

I guess that ipython scans the content of site-packages when it starts (maybe that's one of the reasons it is so much slower to start than the regular python interpreter, and why I don't use ipython) and breaks when it can't access one of the files. This almost sounds like an ipython bug to me, but I won't take care of reporting it

Just to be sure, I have checked if some other files were having the same problem in 2.8.0, but it seems only files in PMP have this problem (you may want to check if I used the correct value for -perm below

```
(cdatm14) [jypmce@ciclad-ng envs]$ find cdatm14 ! -perm /g+r,o+r

cdatm14/share/pmp/obs_info_dictionary.json
cdatm14/share/pmp/disclaimer.txt
cdatm14/share/pmp/demo/pmp_input_parameters_demo1.py
cdatm14/share/pmp/demo/pmp_input_parameters_demo2_save_interpolated_clims.py
cdatm14/share/pmp/default_regions.py
cdatm14/share/pmp/pcmdi_metrics_table
cdatm14/share/pmp/graphics/vcs/portraits.scr
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_0.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/seasonal_mean.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/annual_mean.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/io.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/cor_xyt.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/zonal_mean.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/std_xyt.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/__init__.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/meanabs_xy.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/seasonal_mean.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_xy.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/mean_climate_metrics_calculations.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/cor_xy.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/std_xy.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/annual_mean.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/bias.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/io.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_xyt.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/cor_xyt.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/std_xyt.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/mean_climate_metrics_calculations.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/cor_xy.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/meanabs_xy.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/bias.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_xyt.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_xy.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/rms_0.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/__init__.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/zonal_mean.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/pcmdi/std_xy.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/__init__.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/io/__init__.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/io/__init__.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/io/base.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/io/base.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/version.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/__init__.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/graphics/__init__.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/graphics/portraits.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/graphics/__init__.pyc
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/graphics/portraits.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics/version.py
cdatm14/lib/python2.7/site-packages/pcmdi_metrics-v1.1.1-py2.7.egg-info
```

**Comment by durack1** [comment]
@jypeter thanks for this feedback, when the conda packages are being built we might have to add in a perm wash to ensure that similar issues don't occur in the future.. It's the first time that we've received such a specific use case issue, some other folks using the PMP are doing so in a shared install env

**Comment by jypeter** [comment]
Can you check what access rights you have for this file (and the others), ideally in a 2.8.0 version?

Also, the **date** of the _pcmdi_metrics-v1.1.1-py2.7.egg-info_ file is **21 juil. 2016**, which seems kind of old to me, but I don't know what it should be. Could it be that I got a wrong and outdated version?

**Comment by doutriaux1** [comment]
@jypeter when I install for users here I always do some sort of rechmod to make sure users can read files. I don't think it's due to pmp.

**Comment by jypeter** [comment]
Not due to PMP? Why is it that I have this access rights problem only with PMP (see my find command above), on 2 separate servers? I have installed PMP the same way I have installed other packages. Look for "PMP" in my [2.8.0 installation notes](https://wiki.lsce.ipsl.fr/pmip3/doku.php/other:uvcdat:cdat_conda:cdat_2_8_0#packages_that_have_no_dependency_problems)

Can one of you guys check the access rights of the mentionned PMP file(s) in one of your installation (where you have not done a chmod -R a+rX cdat)

**Comment by doutriaux1** [comment]
@durack1 @zshaheen where is the pmp conda package built at? crunchy and ocenaonly have weird permission mask by dsefault, could that be it?

**Comment by zshaheen** [comment]
@doutriaux1 @durack1 Nightlies are done on Travis. I didn't build the conda packages for v1.1.1, but I usually use aims4, however I never worked on a shared env.

**Comment by durack1** [comment]
This issue was solved in [1.1.2](https://github.com/PCMDI/pcmdi_metrics/milestone/8) so closing
